algopy_testing/tests/artifacts/PrimitiveOps/contract.py

```diff
@@ -138,6 +138,8 @@ def verify_bytes_xor(self, a: Bytes, b: Bytes) -> Bytes:
         return result
 
     @arc4.abimethod()
-    def verify_bytes_not(self, a: Bytes) -> Bytes:
+    def verify_bytes_not(self, a: Bytes, pad_size: UInt64) -> Bytes:
+        a = op.bzero(pad_size) + a
```

Comment:
I assume thats what allows skipping the arg size limit on input parameters?

Reply:
Yes. That is the approach we have taken for getting around the arg size limit on input parameters.